index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Tribute Page</title>
+    <link rel="stylesheet" type="text/css" href="./css/styles.css" />
+  </head>
+  <body>
+    <div id="main">
+      <div id="title"><h1>Dr. Norman Borlaug</h1></div>
+      <div id="img-div">
+        <img id="image" src="./img/arch.jpg" alt="A Building" />
+        <div id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase
+          wheat yields - part of his life-long war on hunger.
+        </div>
+      </div>
+
+      <div id="tribute-info">
+        <h4 id="titles">Here's a time line of Dr. Borlaug's life:</h4>
+        <a id="tribute-link" target="_blank" href=""></a>
+        <ul>
+          <li><span class="bold">1914</span> - Born in Cresco, Iowa</li>
+          <li>
+            <span class="bold">1935</span> - Has to stop school and save up more money. Works in the
+            Civilian Conservation Corps, helping starving Americans. "I saw how food changed them",
```

Comment:
Let's use [an `article` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) here, using this as your sectional element denote that the section is a atomic part of the webpage that could stand alone on it's own, this also makes the part more semantic.